src/components/colorbar/has_colorbar.js

```diff
@@ -0,0 +1,18 @@
+/**
+* Copyright 2012-2015, Plotly, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the MIT license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+
+'use strict';
+
+
+module.exports = function hasColorbar(container) {
```

Comment:
this used to be part of `Plotly.Colorscale`. Makes more sense to put it in `Colorbar`.